app/elements/json-combine/json-combine.html

```diff
@@ -0,0 +1,65 @@
+<link rel="import" href="../polymer/polymer.html">
```

Comment:
This line is currently causing a console error on page load.  It should be `<link rel="import" href="../../bower_components/polymer/polymer.html">`